packages/client/wallets/aa/src/blockchain/wallets/FireblocksNCWallet.ts

```diff
@@ -19,9 +19,24 @@ export const FireblocksNCWallet = async (
     userEmail: string,
     crossmintService: CrossmintService,
     chain: Blockchain,
-    passphrase?: string
+    passphrase?: string,
+    walletId?: string,
+    deviceId?: string
```

Comment:
this would be alot better if it takes an object instead of a list of optional params

then u can make walletId and deviceId both required at the same time